templates/data/postgresql.html

```diff
@@ -1,481 +1,561 @@
-        <li class="p-list__item has-bullet">We use <a href="https://www.pgbouncer.org/">PgBouncer</a> to provide connection pooling and transparent failovers for the connected applications. </li>
-        <li class="p-list__item has-bullet">We expose two endpoints. One for read and write operations on the primary, and another one to scale reads on the replicas. </li>
-        <li class="p-list__item has-bullet">We can backup our deployments to any S3 compatible storage solution.</li>
-        <li class="p-list__item has-bullet">We provide a full monitoring and alerting solution based on Prometheus, Loki and Grafana.</li>
-      </ul>
-      <a class="p-button" href="/blog/postgresql-high-availability">Set up your PostgreSQL database</a>
-    </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-section">
-  <div class="row--25-75 is-split-on-medium">
-    <hr class="p-rule"/>
-    <div class="col">
-      <h2>Services</h2>
+  <section class="p-section u-fixed-width">
+    <hr />
+    <div class="p-section--shallow">
+      <h2>Go further with Charmed PostgreSQL</h2>
     </div>
-    <div class="col">
-      <div class="row">
-        <div class="col-3">
-          <h3 class="p-heading--5">Managed PostgreSQL</h3>
-        </div>
-        <div class="col-6">
-          <p>We manage your PostgreSQL deployments in the cloud of your choice. We can build a private cloud, adapt to your existing one or use public cloud infrastructure.</p>
-          <p>We take care of automatically deploying, hardening, patching, optimising and upgrading your PostgreSQL databases using our <a href="https://charmhub.io/postgresql">open source operators</a>.</p>
+    <div class="p-equal-height-row">
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item u-hide--small u-hide--medium">
+          <div class="p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/3493be78-Kernelt industries-80 1-1.png",
+                        alt="",
+                        width="568",
+                        height="932",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+            }}
+          </div>
+          <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+        </div>
+        <p class="p-equal-height-row__item">
+          Read our tutorials and installation guides to learn how to deploy and manage PostgreSQL within and across public and private clouds.
+        </p>
+        <div class="p-equal-height-row__item">
+          <hr class="is-muted u-hide--medium" />
+          <p>
+            Read the documentation for <a href="https://charmhub.io/postgresql-k8s">K8s</a> and <a href="https://charmhub.io/postgresql/docs/t-overview">VMs</a>
+            &nbsp;&rsaquo;
+          </p>
```

Comment:
```suggestion
            Read the documentation for <a href="https://charmhub.io/postgresql-k8s">K8s</a> and <a href="https://charmhub.io/postgresql/docs/t-overview">VMs&nbsp;&rsaquo;</a>
          </p>
```
Please include the `>` into the hyperlink

Reply:
I would have but I didn't because we have two links here so it seemed as if there was more emphasis on one and not other but if you still think it to be okay I will add it

Reply:
Maybe @Sophie-32 can weigh in on this?

Reply:
Leave out the chevron for this one so its clear these are two separate links please!